b2bAuthorizer/src/app/eventHandler.js

```diff
@@ -121,12 +133,21 @@ async function handleEvent(event) {
         context: {},
         usageIdentifierKey: null
       }
+      if(issuerId) {
+        const metric = prepareMetric("Unknown_iss", decodedJwtToken)
+        metricsHandler.addMetric(metric.metricName, metric.unit, metric.value, metric.dimension, metric.metadata)
+      }
+      const metrics = prepareMetricsJwtData(decodedJwtToken, false)
```

Comment:
@ViVeniero che succede se decodedJwtToken è null? cioè se nel try è proprio getDecodedToken ad andare in eccezione?
La prepareMetricsJwtData non è predisposta a gestire un null.

Reply:
se getDecodedToken va' in eccezione probabilmente non è un AuthenticationError quindi segue l'else di riga 143. Ho in ogni caso predisposto la prepareMetricsJwtData a restituire [] se il decodedJWT è undefined o null